dubbo-serialization/dubbo-serialization-api/src/main/java/org/apache/dubbo/common/serialize/Constants.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.dubbo.common.serialize;
+
+public interface Constants {
```

Comment:
I have a doubt here, why put them in an interface, instead of a class, is there some specific consideration?

Reply:
an interface is perfect place to introduce constants.

Reply:
I have some other thoughts: an interface is a type represents behaviour, if classes accidently implement `Constants`, the constants will be exposed and will pollute the their namespace. 

Just a noninstantiable class will avoid these problems and is better in my opinion, what do you think?